.github/workflows/cpu-inference.yml

```diff
@@ -24,6 +24,8 @@ jobs:
   unit-tests:
     runs-on: [self-hosted, cpu]
 
+    env: {ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true} # Allow using Node16 actions
```

Comment:
@mauryaavinash95 - can you merge the latest DeepSpeed master branch into this dev branch so we can see these merge artifacts resolved now that they are merged into DeepSpeed?